app/build.gradle

```diff
@@ -33,16 +32,16 @@ kotlin {
 
 //Android
 android {
-    compileSdkVersion target_sdk_version
-    buildToolsVersion '28.0.3'
+    compileSdkVersion android_target_sdk
```

Comment:
Above this, the coroutines are as they were still experimental, this part can be removed:
```
kotlin {
    experimental {
        coroutines "enable"
    }
}
```

Reply:
Removed! 1f876b5